src/amo/components/Footer/index.js

```diff
@@ -86,18 +85,6 @@ export class FooterBase extends React.Component {
                   {i18n.gettext('Site Status')}
                 </a>
               </li>
-              <li>
-                <a
-                  href="#desktop"
-                  className="Footer-link Footer-desktop"
```

Comment:
just double checking: it seems fine but are there any tests/styles that need to be updated/removed after this change?

Reply:
There were not any tests covering this currently, so nothing needs to change.